src/components/structure/interacting/navigation/navigation.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useContext } from 'react';
+import { observer } from 'mobx-react-lite';
+
+import Logo from '../../../base/visualizing/logo';
+import CategoryFilter from '../../../base/interacting/category-filter';
+// import Switcher from '~/components/atoms/switcher';
+
+import translations from './translations.json';
+import './navigation.scss';
+
+import StoreContext from '../../../../store/StoreContext';
+
+type Translations = {
+  de: Record<string, string>
+};
+
+const Navigation = () => {
+  const useTranslation = (_: string, translations: Translations) => ( { t: (key: string, _?: Record<string, string>) => translations.de[key] } );
+  const { t } = useTranslation('Navigation', translations);
+  const { globalSearch } = useContext(StoreContext);
+  const isActive = (activeFilter:string, filterValue:string) => { return activeFilter === filterValue ? 'is-active' : '' }
+
+  const navStructure = [
+    {
+      title: 'Prints and Drawings',
+      filterValue: 'GRAPHICS',
```

Comment:
Die Filter-Values sind ja recht API-spezifisch. Auf diese Art und Weise koppelt man sie recht stark an die Navigation-Komponente.

Um dieses Problem zu lösen, würde ich vorschlagen, in der `api/globalSearch.ts` ein Enum einzuführen:

```typescript
export enum EntityType {
  GRAPHICS = 'GRAPHICS',
  PAINTINGS = 'PAINTINGS',
  DOCUMENTS = 'DOCUMENTS',
  UNKNOWN = 'UNKNOWN',
}
```

In dem Code-Schnippsel wird es zudem auch exportiert, damit man es auch außerhalb nutzen kann. Z.B. im GlobalSearch-Store. Das `EntityType.UNKNOWN` wäre dann als Default-Value gedacht.
Dieser kann das Enum erneut exportieren (mit anderem Namen), um es dann über den StoreContext in der `navigation.tsx` nutzen zu können:

```typescript
import StoreContext, { GlobalSearchEntityType } from '../../../../store/StoreContext';

/* ... */

const navStructure = [
  {
    title: 'Prints and Drawings',
    filterValue: GlobalSearchEntityType.GRAPHICS,
  },
  {
    title: 'Paintings',
    filterValue: GlobalSearchEntityType.PAINTINGS,
  },
  {
    title: 'Archival Documents',
    filterValue: GlobalSearchEntityType.DOCUMENTS,
  }
];
```